sale_payment_term_interest/i18n/fr.po

```diff
@@ -42,6 +42,11 @@ msgid "Interest line"
 msgstr "Ligne d'intérêt"
 
 #. module: sale_payment_term_interest
+#: field:account.payment.term,interest_min:0
+msgid "Minimum Interest Amount"
+msgstr "Montant d'intérêt minimum"
+
+#. module: sale_payment_term_interest
 #: model:ir.model,name:sale_payment_term_interest.model_account_payment_term
 msgid "Payment Term"
 msgstr "Condition de règlement"
```

Comment:
In account addon the french translation is "Conditions de règlement " so I guess you are correct